generators/app/templates/LICENSE

```diff
@@ -1,202 +0,0 @@
-
```

Comment:
I was just thinking we may need to keep LICENSE and NOTICE inside the individual `packages/terra-component` directories so they are distributed along with the npm package. 

My thought is have this file and NOTICE generated in same directory as the components README file 
e.g.:
`packages/terra-component/README.md`
`packages/terra-component/LICENSE`
`packages/terra-component/NOTICE`